From bda213f12d241670f8244988ed01b33966b93f6d Mon Sep 17 00:00:00 2001
From: AntoinePrv <AntoinePrv@users.noreply.github.com>
Date: Wed, 13 May 2020 16:40:21 -0400
Subject: [PATCH] Fix libobjscip

---
 scip/src/CMakeLists.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/scip/src/CMakeLists.txt b/scip/src/CMakeLists.txt
index e0c59f3..5542a0b 100644
--- a/scip/src/CMakeLists.txt
+++ b/scip/src/CMakeLists.txt
@@ -933,6 +933,7 @@ if(CXXONLY)
 endif()
 
 add_library(libobjscip EXCLUDE_FROM_ALL ${objscipsources})
+target_link_libraries(libobjscip PUBLIC libscip)
 setLibProperties(libobjscip "objscip")
 
 add_library(libnlpi EXCLUDE_FROM_ALL
@@ -1027,14 +1028,14 @@ install(FILES ${symheaders} DESTINATION include/symmetry)
 install(FILES ${blockmemshellheader} DESTINATION include/blockmemshell)
 
 # install the binary and the library to appropriate lcoations and add them to an export group
-install(TARGETS scip libscip EXPORT scip-targets
+install(TARGETS scip libscip libobjscip EXPORT scip-targets
         LIBRARY DESTINATION lib
         ARCHIVE DESTINATION lib
         RUNTIME DESTINATION bin
         INCLUDES DESTINATION include)
 
 # Add all targets to the build-tree export set
-export(TARGETS scip libscip
+export(TARGETS scip libscip libobjscip
   FILE "${CMAKE_BINARY_DIR}/scip-targets.cmake")
 
 #make soplex and zimpl dir absolute for the config file
-- 
2.26.2

